packages/core/src/navigator/drawer/MenuSeparator.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useMemo} from 'react';
+import {View, StyleSheet} from 'react-native';
+import {Icon, Text, useThemeColor} from '@axelor/aos-mobile-ui';
+
+const MenuSeparator = ({title, iconSize = 15, style}) => {
+  const Colors = useThemeColor();
+
+  const styles = useMemo(() => {
+    return getStyles(Colors);
+  }, [Colors]);
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.childrenContainer}>
+        <View style={[styles.menuItemActive]} />
```

Comment:
I think you can remove it and thus delete the View with childrenContainer style, don't forget to remove styles of deleted components